libenkf/tests/value_export.c

```diff
@@ -0,0 +1,118 @@
+/*
+   Copyright (C) 2017  Statoil ASA, Norway.
+
+   The file 'value_export.c' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+#include <ert/util/test_util.h>
+#include <ert/util/test_work_area.h>
+
+#include <ert/enkf/value_export.h>
+
+void test_create() {
+  test_work_area_type * work_area = test_work_area_alloc("value_export");
+  value_export_type * export = value_export_alloc( NULL, "parameters");
+  test_assert_int_equal( 0 , value_export_size( export ));
+  test_assert_true( value_export_is_instance( export ));
+
+  value_export_txt( export );
+  test_assert_false( util_file_exists( "parameters.txt" ));
+
+  value_export_json( export );
+  test_assert_false( util_file_exists( "parameters.json" ));
+
+  value_export_free( export );
+  test_work_area_free( work_area );
+}
+
+
+
+void test_export_json() {
+test_work_area_type * work_area = test_work_area_alloc("value_export");
+  value_export_type * export = value_export_alloc( "path", "parameters");
+  util_make_path( "path" );
+
+  value_export_append(export, "KEY100", 100);
+  value_export_append(export, "KEY200", 200);
+  test_assert_int_equal( 2 , value_export_size( export ));
+  value_export_json( export );
+  test_assert_true( util_file_exists( "path/parameters.json" ));
+  value_export_free( export );
+  test_work_area_free( work_area );
+}
+
+
+void test_export_txt__() {
```

Comment:
This is not called!

Reply:
So we should either call it or remove it..